### PR TITLE
feat(text field): add action-items slot (VIV-1188)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/text-field/text-field.template.ts [hunk 1]
```diff
@@ -29,7 +30,8 @@ const getStateClasses = ({
 	[`appearance-${appearance}`, Boolean(appearance)],
 	[`shape-${shape}`, Boolean(shape)],
 	['no-label', !label],
-	['success connotation-success', Boolean(successText)]
+	['success connotation-success', Boolean(successText)],
+	['hide-footer', !(actionItemsSlottedContent?.length)]
```
Comment (YonatanKra) [inline libs/components/src/lib/text-field/text-field.template.ts:34]: This one's for hiding the footer, and we have another `hide` for the wrapper?
Anyway, can you please add a test for this too?

File: libs/components/src/lib/text-field/ui.test.ts [hunk 2]
```diff
@@ -34,6 +34,8 @@ test('should show the component', async ({page}: { page: Page }) => {
 
 	await page.waitForLoadState('networkidle');
 
+	//await page.pause();
```
Comment (YonatanKra) [inline libs/components/src/lib/text-field/ui.test.ts:37]: Please remove this line

File: libs/components/src/lib/text-field/text-field.template.ts (hunk 1 above)
Comment (rachelbt) [reply libs/components/src/lib/text-field/text-field.template.ts:34]: this is my stupid copy paste....
I'll remove this one.

**Review by YonatanKra — APPROVED** [review]
Code's solid :)
If @AyalaBu is OK with the result, we can merge this.

**Pull request merged**